### PR TITLE
feat: show fewer details on dashboard cards

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

in this community slack thread https://posthogusers.slack.com/archives/C01GLBKHKQT/p1667323220040869

A user suggests using less dashboard space for information about the insights, leaving more for the insights themselves

## Changes

* extract card meta from the insight card component so it can be used in text cards
* allow bold number cards to have a smaller minimum height
* allow bold number cards to have a larger font
* put more information behind the show details button

### turning the flag on

with-feature-flag

### on a dashboard

minimalist-mode

### bold number font size

bigger-bold-number-text

## How did you test this code?

running locally and 👀

### CODE REVIEW

File: frontend/src/scenes/dashboard/dashboardLogic.tsx [hunk 1]
```diff
@@ -589,7 +591,7 @@ export const dashboardLogic = kea<dashboardLogicType>({
                             w: width,
                             h: h || defaultHeight,
                             minW,
-                            minH: tile.text ? 2 : minH,
+                            minH: tile.text ? 2 : isBoldNumber ? 4 : minH,
```
Comment (Twixes) [inline frontend/src/scenes/dashboard/dashboardLogic.tsx:594]: 👌

**Review by Twixes — CHANGES_REQUESTED** [review]
I like that you went the way of user feedback, but I'm worried this might be too reactive. The changes here are pretty far reaching for what is our core UI. I can see a couple issues:

- The margins around the title area are off:
    Screenshot 2022-11-02 at 09 39 52
    The margin between the top row and the title is also off when the minimalist mode flag is off:
     Screenshot 2022-11-02 at 09 45 02
- I'm pretty positive we should still always be showing insight descriptions on dashboards, they're genuinely important at conveying context…
- …what we perhaps don't _have_ to show at all in cards is "_No description_". When there's no description we could just hide that.
- If we move "Last modified" to details, it should probably be next to "Created by", in the same format ("Last modified by"):
    Screenshot 2022-11-02 at 09 43 45
- Details for text cards make sense, but the margins look off again:
     Screenshot 2022-11-02 at 09 46 53
  Then, the way there's a ton of white space on the left side above the divider and a ton of white space on the right side below the divider doesn't look great to me. What we could do there is put "TEXT" on the left above-divider side (analogous to "FUNNEL • LAST 6 MONTHS" – would be good for visual rhythm), and include both "Created by" and "Last modified by" in the below-divider side.

Overall I agree with the direction, but we should hold ourselves to a really high UX standard here, and make changes as targeted as possible.

Also, some Cypress tests are failing.

**Comment by Aareksio** [comment]
As the user who posted feedback, in relation to @Twixes's critique. 

---

It seems important to determine the target audience:
> [descriptions a]re genuinely important at conveying context

The description is surely useful for first-time visitors, who need to learn the context, but:
a) returning visitors already know the context;
b) good chart title should summarize the context enough for the description to be available as needed via progressive disclosure;

As a returning visitor, I am frustrated to be limited to 6 charts on 4k screen, with space reserved for borderline useless information. My team needs to start the day knowing the stats. They check it every day. Even if there were descriptions, they likely wouldn't notice if it changed, as their brain stopped noticing them long ago.

If you keep making compromises, no group will end up happy.

---

The below is what was suggested in Slack thread, aimed at advanced users:

image

2 points to possibly address, when considering changes:
- "TRENDS", "FUNNEL", "LIFECYCLE" are like "// a string variable" comments - no need to tell me what's below
- "LAST 7 DAYS" is duplicated in dashboard view; Is there a way to change it per-chart, not dashboard?

---

I wholeheartedly agree no changes should be made on a whim. This is only user feedback and should be treated as such.

**Comment by pauldambra** [comment]
> "LAST 7 DAYS" is duplicated in dashboard view; Is there a way to change it per-chart, not dashboard?

@Aareksio if each insight is saved with different date ranges, and the dashboard is set to "custom" for date filter. Then each insight will be shown with it's "local" date range. So you may have a row with "7 days" "30 days" "90 days" next to each other.

If the insights happen to be saved with the same date ranges, or the dashboard has a date range set, then each of them will show the same value.

------

I can take a look at (for posthog cloud) how many insights that are on dashboards also have descriptions set so we can guesstimate the impact of removing it.

**Pull request closed without merging**